test-suite/coq-makefile/timing/precomputed-time-tests/run.sh

```diff
@@ -4,7 +4,8 @@ set -x
 set -e
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
-export COQLIB="$(cd ../../../.. && pwd)"
+COQLIB="$(cd ../../../.. && pwd)"
+export COQLIB
 
 ./001-correct-diff-sorting-order/run.sh || exit $?
 ./002-single-file-sorting/run.sh || exit $?
```

Comment:
Nothing you changed, but isn't || exit $? redundant with set -e?

Reply:
That's true. Will open an issue on Shellcheck.

Reply:
Fixed. Opened [an issue on ShellCheck](https://github.com/koalaman/shellcheck/issues/1163).